.claude/skills/auto-claude-mcp/skill.md

```
@@ -0,0 +1,692 @@
+
+## Direct Start Trigger (File-Based)
+
+Trigger a task to start immediately by setting `status: 'start_requested'`:
+
+```bash
+mkdir -p ".auto-claude/specs/065-my-task"
+echo '{}' > ".auto-claude/specs/065-my-task/task_metadata.json"
+cat > ".auto-claude/specs/065-my-task/implementation_plan.json" << 'EOF'
+{
+  "feature": "My Task",
+  "description": "Task description",
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)